include/net/nrf_cloud_pgps.h

```diff
@@ -239,6 +239,34 @@ int nrf_cloud_pgps_request_all(void);
 #endif /* CONFIG_NRF_CLOUD_MQTT */
 
 #if defined(CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE)
+
+/**@brief If REST is not used, but some new transport, call this function
```

Comment:
`If REST is not used, but some new transport...` perhaps could be reworded to something like `If using a custom transport mechanism`?  Maybe better left for the doc writers :)